README.md

```diff
@@ -1,36 +1,44 @@
 # Fork CMS Catalog module
+
 This catalog module is insipred by the created by Webleads. We upgraded this module from Fork version 3 to version 5. By this we are able to use more of the Fork features.
 
 ## Capabilities
+
 At the moment it is almost the same as the Webleads module, but we left out some code and are still working on this, our changes are:
-* Improved the products by adding:
-  * Related products
-  * Special prices (sale prices)
-  * Added a article number
-  * Added VAT options
-  * Add product specifications on the flow. The user gets the option to create or select one while adding or editing a product
+
+- Improved the products by adding:
```

Comment:
This was done by my Prettier markdown formatter 😅  I can revert if needed

Reply:
No problem, i like it ;-)